index.md

```diff
@@ -6,12 +6,11 @@ Scientific software teams are typically focused on the creation of a new set of
 
 The Productivity and Sustainability Improvement Planning (PSIP) process recognizes that productivity and sustainability improvements for scientific software benefit from an incremental, iterative approach.
 
-Get Started with PSIP:
+_Get Started with PSIP_:
```

Comment:
```suggestion
## Get Started with PSIP
```
This is semantically a header, right?

Reply:
Technically, maybe? I like the idea of making it a header more than its current format. So I accept.